data-raw/process_data_sets_maggie.R

```diff
@@ -0,0 +1,72 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+
+
+
+# nba-carmelo---------------------------------------------------------------------
+#Tame Data Principle 1: naming conventions
+nba_carmelo <- read.csv("data-raw/nba-carmelo/nba_elo.csv")
+colnames(nba_carmelo) <- colnames(nba_carmelo) %>%
+  tolower() %>%
+  str_replace_all(" ", "_")
+  #converting variable names to snake_case
+#Principle 2: variables identifying observational units
+  #observational units are specific games, no changes seem necessary
+#Principle 3: dates
+  #there is already a date variable in year-month-day format
+#Principle 4: Factors, Characters, Logicals
+  #there are 20 variables. 14 of them- all the elo, carmelo, and score variables- are quantitative not categorical so we can leave them as is. 
+  #season is an ordinal factor (I think), teams are regular factors, playoff and neutral are TRUE/FALSE (binary logical variables), date I assume is fine as is. 
+nba_carmelo <- nba_carmelo %>%
+  mutate(
+  team1 = as.factor(team1),
+  team2 = as.factor(team2),
+  season = factor(season, levels=c(1947:2018), ordered=TRUE),
+  playoff = ifelse(playoff == "t", TRUE, FALSE),
+  neutral = ifelse(neutral == 1, TRUE, FALSE))
+#Principle 5: tidy data format
+  #data appears to adhere to guidelines and is in "long/narrow" format
+  #maybe could be narrower if separated elo_score and carmelo_score then also had type (1_pre, 2_pre, prob1, prob2, 1_post, 2_post) but I'm not sure if this is as intuitive
```

Comment:
I agree, I'm not sure what the answer is at this point; but let's address these all in one go once you've made a first pass thru all the data. Let's have the following convention for "let's decide later": Mark any pending questions with `TODO: WRITE ISSUE TO DISCUSS LATER` as comment.